src/AccessibilityInsights.SharedUx/Controls/HierarchyControl.xaml

```diff
@@ -89,7 +89,7 @@
             </Grid.ColumnDefinitions>
             <Border Height="24" Margin="8,5,8,5" Background="{DynamicResource ResourceKey=LightGreyBrush}">
                 <DockPanel>
-                    <controls:FabricIconControl DockPanel.Dock="Left" Margin="5,0,5,0" GlyphName="Search" FontSize="{StaticResource ConstStandardTextSize}" GlyphSize="Custom" HorizontalAlignment="Center" VerticalAlignment="Center" Foreground="{DynamicResource ResourceKey=IconBrush}"/>
+                    <controls:FabricIconControl DockPanel.Dock="Left" Margin="5,0,5,0" GlyphName="Search" FontSize="{StaticResource ConstStandardTextSize}" GlyphSize="Custom" HorizontalAlignment="Center" VerticalAlignment="Center" Foreground="{DynamicResource ResourceKey=IconBrush}" ShowInControlView="False"/>
```

Comment:
![search icon in hierarchy tree](https://user-images.githubusercontent.com/7775527/64988722-f804fa00-d880-11e9-8e56-2514f5601609.png)